spec/spec_helper.rb

```diff
@@ -17,10 +17,23 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 
+require "active_support/all"
+require "active_support/testing/time_helpers"
+
+class Rails
```

Comment:
@stevo It was a bit tricky, as `Rails` have to be declared before `Calculation` is required, and `Rails` have to actually implement `cache` method before it can be stubbed.
I was able to achieve it by:
- moving ```require "rails-patterns"``` to the end of the file,
- using `config.before(:all)`,
- requiring `rspec/mocks/standalone` to allow for mocking outside of the per-test lifecycle, and
- declaring a double instead of stubbing.

But, I think that a simple class declaration here was a little bit more comprehensible solution :)